pom.xml

```diff
@@ -139,8 +139,8 @@
         <sparkshim.module.name>spark32</sparkshim.module.name>
         <sparkshim.artifactId>spark-sql-columnar-shims-spark32</sparkshim.artifactId>
         <spark.version>3.2.2</spark.version>
-	<iceberg.version>1.3.1</iceberg.version>
-	<delta.package.name>delta-core</delta.package.name>
+        <iceberg.version>1.3.1</iceberg.version>
```

Comment:
Seems the change should only be needed for CH backend.  Why diff exists here?

Reply:
this is a small style fix, before it looks like:
<img width="772" alt="image" src="https://github.com/apache/incubator-gluten/assets/12025282/a6a9ed56-5333-4148-a8dc-c1b4d69161c9">

Reply:
Oh, didn't notice this. Thanks for change.